app/views/posts/new.html.erb

```diff
@@ -0,0 +1,16 @@
+
+
+<%= form_with model: @comment, url:new_user_post_comment , method: :post do |form| %>
+<h3>Create new Post</h3>
+  <div class="mb-3">
+    <%= form.text_field :title, class: 'form-control', placeholder: 'Title' %>
+  </div>
+  <div class="mb-3">
+    <%= form.text_area :text, class: 'form-control', placeholder: 'Content' %>
+  </div>
+
+  <div class="mb-3">
+    <%= form.submit 'Create post', class: 'btn btn-primary' %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] Nice job on adding the form however I can see an error accruing when I want to access the form for creating a new post. Kindly solve them.
- [ ] Also I think It would be better If you add a button that links to this form in your UI